jubatus/server/framework/mixer/linear_mixer.cpp

```diff
@@ -365,24 +368,28 @@ void linear_mixer::stabilizer_loop() {
         return;
       }
 
+      // Release the lock and wait for signal or 0.5 secs.
```

Comment:
``signal`` is ``notify`` function in https://github.com/jubatus/jubatus/blob/0.9.3/jubatus/server/framework/mixer/linear_mixer.cpp#L342.

``the condition`` means ``signal or 0.5 secs``

I changed this comment because it's confusing.